src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -191,6 +188,14 @@ default void log(
         return Optional.empty();
     }
 
+    default ResponseEntity<Problem> fallback(
+            @SuppressWarnings("UnusedParameters") final Throwable throwable,
+            @SuppressWarnings("UnusedParameters") final Problem problem,
+            @SuppressWarnings("UnusedParameters") final NativeWebRequest request,
+            @SuppressWarnings("UnusedParameters") final HttpHeaders headers) {
+        return ResponseEntity.status(NOT_ACCEPTABLE).body(null);
```

Comment:
We are currently investigating whether this fallback works in all servlet container envs - I think there is an issue with closing the response if no body is given.